menu/Menu.java

```diff
@@ -0,0 +1,229 @@
+                .addGap(32, 32, 32)
+                .addComponent(sair, javax.swing.GroupLayout.PREFERRED_SIZE, 47, javax.swing.GroupLayout.PREFERRED_SIZE)
+                .addGap(47, 47, 47)
+                .addComponent(teste)
+                .addContainerGap(javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
+        );
+
+        javax.swing.GroupLayout layout = new javax.swing.GroupLayout(getContentPane());
+        getContentPane().setLayout(layout);
+        layout.setHorizontalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addComponent(Fundo, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                .addContainerGap())
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(javax.swing.GroupLayout.Alignment.TRAILING, layout.createSequentialGroup()
+                .addComponent(Fundo, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                .addContainerGap())
+        );
+
+        pack();
+        setLocationRelativeTo(null);
+    }
+
+    /* Configurando ação do botão iniciar */
+    private void iniciarActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_iniciarActionPerformed
+        
+        var nome = inputNome.getText();
+        teste.setText("Olá " + nome + "!");
+        
+        /*// Configurando Tabuleiro
+        JFrame frame = new JFrame();
+        frame.setTitle("Engenharia do Xadrez part. II");
+        frame.getContentPane().setBackground(Color.black);
+        frame.setLayout(new GridBagLayout());
+        frame.setMinimumSize(new Dimension(1080, 720));
+        frame.setLocationRelativeTo(null);
+        frame.setDefaultCloseOperation(frame.DISPOSE_ON_CLOSE);
+
+        // Criando e adicionando o tabuleiro ao JFrame 
+        Tabuleiro tabuleiro = new Tabuleiro();
+        frame.add(tabuleiro);
+
+        // Configurando a visibilidade do JFrame
+        frame.setVisible(true);/* */
+    }//GEN-LAST:event_iniciarActionPerformed
+
+    /* Configurando ação do botão sair */
+    private void sairActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_sairActionPerformed
+        JFrame confirmacao = new JFrame("Deseja realmente sair?");
+        String[] opcao = new String[2];
+        opcao[0] = "Sim";
+        opcao[1] = "Não";
+        var escolha = JOptionPane.showOptionDialog(null, "Deseja realmente sair?", "Engenharia de Xadrez part. II", 0, JOptionPane.INFORMATION_MESSAGE, null, opcao, null);
+        if (escolha == 0) {
+            System.exit(0);
+        }
+    }
+
+    private void sairMouseEntered(java.awt.event.MouseEvent evt) {
+        // TODO
+    }
+
+    private void inputNomeActionPerformed(java.awt.event.ActionEvent evt) {
+        // TODO 
+    }
```

Comment:
import aqui também
Falta implementar mesmo?
Se não for implementar pode tirar os métodos